tests/integration/test_data_rows.py

```diff
@@ -475,19 +475,26 @@ def create_data_row(data_rows):
         CUSTOM_TEXT_SCHEMA_NAME].uid
 
 
-def test_create_data_rows_with_invalid_metadata(dataset, image_url):
+def test_create_data_rows_with_invalid_metadata(dataset, image_url,
+                                                is_adv_enabled):
     fields = make_metadata_fields()
     # make the payload invalid by providing the same schema id more than once
     fields.append(
-        DataRowMetadataField(schema_id=TEXT_SCHEMA_ID, value='some msg'))
+        DataRowMetadataField(schema_id=TEXT_SCHEMA_ID, value="some msg"))
 
     task = dataset.create_data_rows([{
         DataRow.row_data: image_url,
         DataRow.metadata_fields: fields
     }])
     task.wait_till_done()
-    assert task.status == "FAILED"
-    assert len(task.failed_data_rows) > 0
+    if is_adv_enabled:
```

Comment:
@kkim-labelbox another difference in ADV
I documented this [here](https://labelbox.atlassian.net/wiki/spaces/AIENG/pages/2240020512/Known+Breaking+Changes+From+Pre-ADV+to+ADV)

Reply:
Is this because ADV partially fails now when there's an error with validation of metadata on one data row?

Reply:
It partially _succeeds_ instead. What can be imported, will be imported. The rest will be reported as failures.